docs/website/src/page/articles-layout.less

```diff
@@ -22,4 +23,14 @@
     color: @nav-accent;
     margin-bottom: 40px;
   }
+
+  &-content {
+    margin-bottom:5px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
```

Comment:
It's the legacy flexbox alias, why do we need it here?

Reply:
Sorry, didn't know that everything is awesome in CSS ))
I see that the `-webkit-line-clamp` works only with the legacy flexbox standard, so no more concerns with it